packages/daimo-api/src/network/viemClient.ts

```diff
@@ -258,17 +241,16 @@ export class ViemClient {
         `[VIEM] tx ${localTxId} ${elapsedMs()}ms: got nonce ${this.nextNonce}`
       );
 
-      // Override nonce with nonce estimate. Optimistically assumes that
-      // all pending transactions will succeed.
-      args.nonce = this.nextNonce + this.pendingTxCount;
```

Comment:
this line was missing an important log. see log above ("...got nonce: ") logging an incorrect nonce, vs the one we're submitting, and not logging `pendingTxCount`.